flow/queue/exceptions.py

```diff
@@ -0,0 +1,42 @@
+class InvalidAid(Exception):
```

Comment:
Because the exception handling is not handled in the exception object.

I use them in almost every file

https://github.com/baxter-cs/roastme/blob/71d5346b08a079db4b9e90a318f6b228c4837c22/flow/queue/db_functions.py#L264